docs/reference/models/image_chat.md

```diff
@@ -0,0 +1,3 @@
+# Chat Models
```

Comment:
Rename it to Image Chat Models. Now there are two pages with the same name in the docs.

Reply:
Fixed